cea/demand/refrigeration_loads.py

```diff
@@ -33,32 +34,28 @@ def has_refrigeration_load(bpr):
     else:
         return False
 
+
 def calc_Qcre_sys(bpr, tsd, schedules):
     # calculate refrigeration loads
-    tsd['Qcre'] = schedules['Qcre'] * bpr.internal_loads['Qcre_Wm2']
+    tsd['Qcre'] = schedules['Qcre'] * bpr.internal_loads['Qcre_Wm2'] * -1.0  # cooling loads are negative
     # calculate distribution losses for refrigeration loads analogously to space cooling distribution losses
     Y = bpr.building_systems['Y'][0]
     Lv = bpr.building_systems['Lv']
-    Qcre_d_ls = ((tsd['Tcdata_sys_sup'] + tsd['Tcdata_sys_re']) / 2 - tsd['T_ext']) * (
+    Qcre_d_ls = ((T_C_REF_SUP_0 + T_C_REF_RE_0) / 2 - tsd['T_ext']) * (
                 tsd['Qcre'] / np.nanmin(tsd['Qcre'])) * (Lv * Y)
     # calculate system loads for data center
     tsd['Qcre_sys'] = tsd['Qcre'] + Qcre_d_ls
 
-    def function(Qcre_sys):
-        if Qcre_sys > 0:
-            Tcref_re_0 = 5
-            Tcref_sup_0 = 1
-            mcpref = Qcre_sys/(Tcref_re_0-Tcref_sup_0)
-        else:
-            mcpref = 0.0
-            Tcref_re_0 = 0.0
-            Tcref_sup_0 = 0.0
-        return mcpref, Tcref_re_0, Tcref_sup_0
-
-    tsd['mcpcre_sys'], tsd['Tcre_sys_re'], tsd['Tcre_sys_sup'] = np.vectorize(function)(tsd['Qcre_sys'])
+    # writing values to tsd, replacing function and np.vectorize call with simple for loop
```

Comment:
Is there a reason for this refactor? The loop takes longer than vectorizing, but it might help readability. (The histogram belows shows the ratio between the new run time and the old run time after running that function 1000 times)
![Figure_1](https://user-images.githubusercontent.com/18548065/58480304-6864ff80-815a-11e9-9c49-716d0ff31795.png)

Reply:
The PR works as desired: thanks @gabriel-happle!
Not sure about the above refactor, though: the new version is more readable, the old version is faster (though both take less than 1s). Maybe @daren-thomas can provide feedback.